packages/shared-http-pieces/package.json

```diff
@@ -37,6 +37,7 @@
 		"@sentry/node": "^7.54.0",
 		"@skyra/http-framework": "workspace:^",
 		"@skyra/http-framework-i18n": "workspace:^",
+		"@skyra/influx-utilities": "workspace:^",
```

Comment:
Honestly speaking, if Influx is to be included, it should also include the pieces.

I was also thinking of having `/shared-influx-pieces` as to not bloat this package too much, and allow opt-in or even using it in Skyra without the need for HFx.

Reply:
Je comptais au passage crée les événement et les commandes ce qui explique cette dépendances